examples/source/1.components/amp-bind-recaptcha.html

```diff
@@ -0,0 +1,113 @@
+<!---
+
+author: elisameyer
+
+--->
+<!--
+  ## Introduction
+
+  This example shows how to create recaptcha input using just amp-bind.
+-->
+
+<!doctype html>
+<html ⚡="">
+  <head>
+    <title>amp-bind-recaptcha</title>
+    <meta charset="utf-8" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <link rel="canonical" href="<% canonical %>">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link rel="canonical" href="https://www.xyz.com" />
+    <!-- import `amp-bind` so recaptcha can have multiple states -->
+    <script
+      async
+      custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
+    ></script>
+    <!-- recaptcha input used to verify user for `amp-form` -->
+    <script
+      async
+      custom-element="amp-form"
+      src="https://cdn.ampproject.org/v0/amp-form-0.1.js"
+    ></script>
+    <!-- stylesheet used to import refresh image for recaptcha -->
```

Comment:
can we please not recommend using font-awesome for a single icon. It's massive. Suggestion: embed an icon as inline SVG (e.g. from here https://material.io/resources/icons/?search=re&icon=autorenew&style=baseline)

Reply:
updated - now using inline img tag with material.io svg icon